model/convert/service_event.go

```diff
@@ -760,32 +755,14 @@ func convertClusterQCVoteData(cdcClusterQCVoteData []cadence.Value) ([]flow.Clus
 			voterIDs = append(voterIDs, voterID)
 		}
 
-		// gather all the vote signatures
-		signatures := make([]crypto.Signature, 0, len(cdcRawVotes.Values))
-		for _, cdcRawVote := range cdcRawVotes.Values {
-			rawVoteHex, ok := cdcRawVote.(cadence.String)
-			if !ok {
-				return nil, invalidCadenceTypeError(
-					"clusterQC[i].vote",
-					cdcRawVote,
-					cadence.String(""),
-				)
-			}
-			rawVoteBytes, err := hex.DecodeString(string(rawVoteHex))
-			if err != nil {
-				return nil, fmt.Errorf("could not convert raw vote from hex: %w", err)
-			}
-			signatures = append(signatures, rawVoteBytes)
+		cdcVoteSignatures, err := getField[cadence.String](fields, "voteSignatures")
+		if err != nil {
+			return nil, fmt.Errorf("failed to decode clusterQCVoteData struct: %w", err)
 		}
-		// Aggregate BLS signatures
-		aggregatedSignature, err := crypto.AggregateBLSSignatures(signatures)
+
+		rawVoteBytes, err := hex.DecodeString(string(cdcVoteSignatures))
```

Comment:
```suggestion
		aggregatedSignature, err := hex.DecodeString(string(cdcVoteSignatures))
```
Extending Yurii's suggestion, keeping the original name here is also clearer that it is a single aggregated signature.